database/testMongoInsert.js

```diff
@@ -21,4 +21,6 @@ let newListing = new Listing(listingProps);
 newListing.save((err, listing) => {
   console.log(err);
```

Comment:
I'm not sure what the output is looking like for you right now, but it might be helpful to add `if (err)` and `else` cases so that there's only one console log, corresponding to whether there was an error or not. Are you seeing `undefined` console logging right now when this is invoked?

Reply:
You can also consider using `console.error` instead of `console.log` for errors.

Reply:
Good call. I was getting null. I've updated.